bayesian_cnn_prometheus/preprocessing/preprocessing_links/create_chunks.py

```diff
@@ -20,12 +20,12 @@ def run(self, global_config: Dict[str, str], array: np.array) -> Generator:
             return chunk_generator
 
     @staticmethod
-    def _create_chunks(data_subset: np.array, chunk_size: tuple = (32, 32, 16)) -> np.array:
+    def _create_chunks(data_subset: np.array, chunk_size: tuple = (32, 32, 16)) -> Generator:
```

Comment:
Może `Generator[np.ndarray, None, None]` , żeby był wiadomy typ tego co on wypłuwa?

Reply:
A tak właściwie to dlaczego [np.ndarray, None, None] a nie np.ndarray?

Reply:
Dlatego, że `Generator` to typ generyczny, a jego [sygnatura](https://docs.python.org/3/library/typing.html#typing.Generator) wygląda tak

![image](https://user-images.githubusercontent.com/21079319/140093295-259c62f2-1c17-45f4-9df8-cd438faa93ac.png)


przykład jest trochę mylący ponieważ niewidać z niego czego dotyczy `SendType`.
Jest to typ argumentu który trzeba przekazać do generatora, żeby on był w stanie zwrócić w jakiś sposób na tym podziałać

[Tu](https://stackoverflow.com/a/26217712/7516134) jest to dobrze wytłumaczone

Reply:
Moglibyśmy w ogóle użyć typ `Iterator[np.ndarray]` tak jak oni mówią w tym ostatnim przykładzie
To by było kozacko

Reply:
Albo nie 
![image](https://user-images.githubusercontent.com/21079319/140094457-b900bdec-9310-4d74-96c2-c631c38427eb.png)